.github/workflows/check.yaml

```diff
@@ -3,112 +3,127 @@
                 run: npm install
-            -
-                name: Remove $HOME\.apify directory on Windows
+
+            -   name: Remove $HOME\.apify directory on Windows
                 if: runner.os == 'Windows'
                 run: Remove-Item -Recurse -Force -ErrorAction SilentlyContinue $HOME\.apify
-            -
-                name: Remove ~/.apify directory on Linux
+
+
+            -   name: Remove ~/.apify directory on Linux
                 if: runner.os != 'Windows'
                 run: rm -rf ~/.apify
-            -
-                name: Run Tests
+
+            -   name: Run Tests
                 env:
                     TEST_USER_TOKEN: ${{ secrets.APIFY_TEST_USER_API_TOKEN }}
                     APIFY_CLI_DISABLE_TELEMETRY: 1
                 run: npm test
 
     test_python_support:
         name: Test Python template support
-        needs: [build_and_test]
         strategy:
+            fail-fast: false
             matrix:
-                os: [ubuntu-latest, windows-latest]
-                python-version: ["3.8", "3.9", "3.10", "3.11"]
+                os: [ ubuntu-latest, windows-latest ]
+                python-version: [ "3.8", "3.9", "3.10", "3.11" ]
         runs-on: ${{ matrix.os }}
 
         steps:
-            -
-                uses: actions/checkout@v3
-            -
-                name: Use Node.js 16
-                uses: actions/setup-node@v3
-                with:
-                    node-version: 16
-            -
-                name: Load cached node_modules
-                uses: actions/cache@v3
+            -   uses: actions/checkout@v4
+
+            -   name: Use Node.js 20
+                uses: actions/setup-node@v4
                 with:
-                    path: |
-                        node_modules
-                        build
-                    key: cache-${{ github.run_id }}
-            -
-                name: Set up Python ${{ matrix.python-version }}
+                    node-version: 20
+                    cache: 'npm'
+                    cache-dependency-path: 'npm-shrinkwrap.json'
+
+            -   name: Install Dependencies
+                run: npm ci --force
+
+            -   name: Set up Python ${{ matrix.python-version }}
                 uses: actions/setup-python@v4
                 with:
                     python-version: ${{ matrix.python-version }}
-            -
-                name: Run Python tests
+
+            -   name: Run Python tests
                 env:
                     TEST_USER_TOKEN: ${{ secrets.APIFY_TEST_USER_API_TOKEN }}
                     APIFY_CLI_DISABLE_TELEMETRY: 1
                 run: npm run test-python
 
+    docs:
+        name: Docs build
+        if: ${{ !contains(github.event.head_commit.message, '[skip ci]') }}
+        runs-on: ubuntu-latest
+        steps:
+            -   name: Checkout Source code
+                uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
```

Comment:
Why the fixed commit here?

Reply:
some time ago renovate did this change for some reason in our repositories, this is just a bad copypasta from other repo that had this problem, will use the tag instead (I am also gradually reverting this in other repos, did this yesterday for crawlee and js sdk too).